.github/workflows/build_python_wheels.yml

```diff
@@ -38,24 +38,29 @@ jobs:
           path: audit_output/
   test:
     needs: build
-    container:
-      image: python:${{ matrix.config.version }}
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.config.experimental }}
     strategy:
+      fail-fast: false
       matrix:
         config:
-        - { version: "3.6", experimental: false }
-        - { version: "3.7", experimental: false }
-        - { version: "3.8", experimental: false }
-        - { version: "3.9", experimental: false }
-        - { version: "3.10", experimental: false }
+        - { python_version: "3.6", artifact_version: "3.6" }
+        - { python_version: "3.7", artifact_version: "3.7" }
+        - { python_version: "3.8", artifact_version: "3.8" }
+        - { python_version: "3.9", artifact_version: "3.9" }
+        - { python_version: "3.10", artifact_version: "3.10" }
+        - { python_version: "pypy-3.7", artifact_version: "3.7" }
```

Comment:
whats the difference between `3.7` and `pypy-3.7`?

Reply:
pypy is a different python interpreter implementation https://www.pypy.org/